js/layout-javascript/simple-list-code.js

```diff
@@ -458,6 +519,13 @@ DynamicList.prototype.attachObservers = function() {
     .on('hide.bs.collapse', '.simple-list-filters-panel .panel-collapse', function() {
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
+    .on('click keydown', '.simple-list-filters-panel', function(event) {
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
+
+      $(event.target).find('.collapse').collapse('toggle');
+    })
     .on('click', '.simple-list-comment-holder', function(event) {
```

Comment:
Why does this not need `keydown` event like the `.news-feed-comment-holder` selector needed?

Reply:
@tonytlwu fixed, i dont understand why this code was lost after the revert.